### ISSUE TITLE
add psrecord to kernel build steps

### ISSUE THREAD

**Issue by flynneva** [ros-realtime/rt-kernel-docker-builder#27]
[psrecord](https://github.com/astrofrog/psrecord) is a really useful tool to measure cpu and memory usage. right now its not included by default in the build but I'd really like it to be since I am using it over on the [`autoware_reference_system`](https://github.com/ros-realtime/reference-system/pull/33) 😄

if there is pushback, not a big deal we can just install it later with `python3 -m pip install psrecord`

**Comment by razr** [comment]
@flynneva how it is related to the kernel build? could you help me to understand where do you want to include it?

**Comment by flynneva** [comment]
@razr i was thinking of trying to include it if we end up providing an entire image, but it's ok I can just add it as a step to the other repo

**Comment by razr** [comment]
I agree we need to keep somewhere extra ```deb``` packages that we need to add to the pristine Ubuntu image. Maybe in the plain ```pkglist``` file? 
```
sudo apt-get install -y $(cat pkglist)
```

**Comment by carlossvg** [comment]
@flynneva I propose to create a customized image including all the tools required for the RTWG benchmarking (reference system, perf test, pendulum_control) and other additional tools that could be useful but might not be required for other users. 

@flynneva What do you think?
